### DISCUSSION TITLE
How to parse raw request/response into `http.Request/Response` in script?

### DISCUSSION THREAD

**Discussion by Unintendedz** [mitmproxy/mitmproxy#6110] [Q&A]
I already know how to serialize flow into raw text like this:
```
from mitmproxy.addons.export import raw_request

def request(flow):
    print(raw_request(flow))
```

but how can I parse raw text into flow?

**Comment by mhils** [comment]
I don't think we have a really good answer to this at the moment. An `import` command would be awesome though!
